python/python/glide/async_commands/server_modules/json.py

```diff
@@ -214,6 +226,107 @@ async def forget(
     )
 
 
+async def strappend(
+    client: TGlideClient,
+    key: TEncodable,
+    value: TEncodable,
+    path: Optional[TEncodable] = None,
+) -> TJsonResponse[int]:
+    """
+    Appends the specified `value` to the string stored at the specified `path` within the JSON document stored at `key`.
+
+    See https://redis.io/commands/json.strapped/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        value (TEncodable): The value to append to the string. Must be wrapped with single quotes. For example, to append "foo", pass '"foo"'.
```

Comment:
"Must be wrapped with single quotes" - this is a stupid requirement for the command :( 
Can we add a single quote around the string if one doesn't exist?  Seems like this is the only command that has this dumb requirement.

Reply:
I dont know, it is annoying I agree